src/components/legend.ts

```diff
@@ -326,6 +329,38 @@ export module Components {
       }
     }
 
+    /**
+     * Gets the function determining the opacity of the symbols of the Legend.
+     *
+     * @returns {(datum: any, index: number) => number}
+     */
+    public symbolOpacity(): (datum: any, index: number) => number;
```

Comment:
Mmm good point. I chose to do it this way because I noticed that accessors in plots are always converted to functions:

```
> var rect = new Plottable.Plots.Rectangle().x(5);
> rect.x().accessor
function (){return n}
```

This would be consistent with d3 (`Plot._bind()` calls `d3.functor()`, after all) but might be confusing to the first time user. What do you think?